Bottom: a4600e0d04beab645b3a1e3b9b529eae2ac6cd6c
Top:    87fe6779f7e24c69511dfda450d99d9f1271edd0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 20:19:06 +0000

Disable perturbation attempt during intersection


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 88f8369..0d620e6 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1158,6 +1158,7 @@ seg_in_seg_arc_line (struct info *i, struct seg *s1, struct seg *s2)
 
   printf ("  Intersect count is %i\n", cnt);
 
+#if 0
   if (cnt == 0)
     {
       printf ("Trying with a perturbation\n");
@@ -1172,6 +1173,10 @@ seg_in_seg_arc_line (struct info *i, struct seg *s1, struct seg *s2)
 
       printf ("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! <<<<<<\n");
     }
+#endif
+
+  if (cnt == 0)
+    return 0;
 
   if (cnt == 2)
     {
